src/azure-cli/setup.py

```diff
@@ -138,7 +140,9 @@
     'websocket-client~=0.56.0',
     'xmltodict~=0.12',
     'javaproperties==0.5.1',
-    'jsondiff==1.2.0'
+    'jsondiff==1.2.0',
+    'azure-mgmt-core~=1.2.0',
```

Comment:
`azure-mgmt-core` and `azure-core` are already in azure-cli-core dependencies, we can remove them here.

Reply:
Removed